src/bloqade/builder/field.py

```diff
@@ -64,99 +130,292 @@ def location(
         labels: Union[List[int], int],
         scales: Union[List[ScalarType], ScalarType, None] = None,
     ) -> "Location":
-        """Address a single atom (or multiple) atoms.
+        """Address a a subset of atoms as part of defining the spatial modulation component of a drive
+        with a unique scaling factor for each of them.
+
+        Args:
+            labels (Union[List[int], int]): A list of site indices or a single site index to target.
+            scales (Union[List[ScalarType], ScalarType, None]): A list of scaling factors for each site or a single scaling factor to target all sites.
+
+        Returns:
```

Comment:
This is a mix of RST (numpy style doc) and Markdown, we should stick with Markdown syntax.

Reply:
Could you give an example of a pure markdown format for the above? I thought this format was markdown compatible (I believe I'm following Google style here). RST style looks drastically different from this according to the following: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

Reply:
The google style is not in markdown? E.g Args needs to be specified as a title instead of indent?

Reply:
more specifically, the `Args:` syntax is from RST (see from your link). In markdown, this should be `### Args` (pick a title number here, I usually use the <h3> tag).

Reply:
here is a screenshot from vscode hint, since it's not markdown you lost the highlights

<img width="965" alt="image" src="https://github.com/QuEraComputing/bloqade-python/assets/8445510/188b7322-f487-4c6f-b71c-e4830d2e7f6a">

Reply:
Ah I understand now. The problem is if I introduce the markdown `###` the rendering on MkDocs starts to fall apart:

<img width="651" alt="Screenshot 2024-05-01 at 8 53 47 AM" src="https://github.com/QuEraComputing/bloqade-python/assets/32608115/2f15dbfb-dfcc-49d5-b932-8fc66c70aa7c">

versus what it looks like without the `###` 
<img width="638" alt="Screenshot 2024-05-01 at 9 00 09 AM" src="https://github.com/QuEraComputing/bloqade-python/assets/32608115/c220ccc3-5255-46ba-b001-829415b8fbf3">

Reply:
maybe should be https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#definition-lists ?

Reply:
@Roger-luo The `???` isn't creating...I don't think VSCode highlights `Args` and `Returns` just try it out on a simple example.

Reply:
Because it's not markdown, vscode only highlights markdown titles. Which is why I'm asking for changing it to title instead of RST here.